coremltools/converters/mil/frontend/torch/test/test_examples.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2022, Apple Inc. All rights reserved.
```

Comment:
nit: For this newly added test file, I'm wondering if we plan to add all test cases used in coremltools docs here? Or do we want to re-use test_conversion_api.py?

Reply:
I think adding unit test for all the examples is a great idea. Could you create a GitHub issue for that?